R/helper_clean_model_data.R

```diff
@@ -44,6 +44,17 @@ clean_model_data <- function(data, datargs, estimator = "") {
   for (da in to_process) {
     name <- sprintf(".__%s%%%d__", da, sample.int(.Machine$integer.max, 1))
     m_formula_env[[name]] <- eval_tidy(mfargs[[da]], data = data)
+
+    if (is_formula(m_formula_env[[name]])) {
```

Comment:
I think this affects all args, not just weights and clusters? Not sure if that is bad though. Maybe move outside loop like fixed effects?

Reply:
This may be moot based on the comments below, but this is a good point. I don't think this will have negative consequences, since none of the arguments that go through this loop should evaluate to a formula, but there may be a speed penalty from adding an unnecessary step to every loop.